CHANGELOG.md

```diff
@@ -1,13 +1,45 @@
-1.0.0-preview4 (In progress)
-=============================
+# 1.0.0-preview4 (In Progress)
 
-Bug Fixes
----------
+## New Features
+
+* ### `LowAudioLevel` detection and warning in `InputTest`.
+
+  During the runtime of the test, if low audio levels are detected from the captured input device, the test will emit a `warning` event. The test will emit a `warning-cleared` event when nominal audio levels are detected only after the `low-audio-level` `warning` has been fired.
+
+  After the `low-audio-level` warning has been raised, it will not be raised again until it has been cleared.
+
+  Example usage:
+  ```ts
+  import { InputTest, testInputDevice, WarningName } from '@twilio/rtc-diagnostics';
+
+  const inputDeviceTest = testInputDevice();
+
+  inputDeviceTest.on(InputTest.Events.Warning, (warning: WarningName) => {
+    // The application can listen for specific warnings...
+    if (warning === WarningName.LowAudioLevel) {
+      // update the ui to show the input device may not be working
+    }
+
+    // ...or access all active warnings.
+    inputDeviceTest.activeWarnings.values().forEach(...);
+  });
+
+  inputDeviceTest.on(InputTest.Events.WarningCleared, (warningType: WarningName) => {
```

Comment:
Here we are using `(warningType: WarningName) => {`. But the example above is using `(warning: WarningName) => {`. Can we make it consistent? I think `(warningName: WarningName) => {` seems better?